src/core/less/structure/page.less

```diff
@@ -0,0 +1,45 @@
+// Page styling
+// ============
+
+.page {
+    background-color: @background-color;
+
+    .page-header {
+        background-color: @background-color-inverted;
+    }
+
+    .page-header-inner {
+        padding: @page-padding-top 0 @page-padding-bottom;
+
+        @media all and (max-width: (@device-width-large + 20px)) {
```

Comment:
all of the media queries like the ones in this file are additional styling for a specific theme specific look, and should be removed

Reply:
Done.